components/automate-cli/pkg/verifyserver/server/api/v1/handler.go

```diff
@@ -20,6 +21,7 @@ type Handler struct {
 	HardwareResourceCountService hardwareresourcecount.IHardwareResourceCountService
 	SoftwareVersionService       softwareversionservice.ISoftwareVersionService
 	S3ConfigService   s3configservice.IS3Config
```

Comment:
Format the s3configservice in same indentation

Reply:
Done